rejoiner/src/main/java/com/google/api/graphql/rejoiner/ProtoToGql.java

```diff
@@ -264,4 +215,32 @@ static String getReferenceName(GenericDescriptor descriptor) {
   static GraphQLTypeReference getReference(GenericDescriptor descriptor) {
     return new GraphQLTypeReference(getReferenceName(descriptor));
   }
+
+  //  private java.util.function.Function<Object, Object> getter(Class clazz, String methodName) {
```

Comment:
This looks interesting, what is this?

Reply:
I should probably remove this.
I came along this article: https://dzone.com/articles/java-reflection-but-faster
LambdaMetafactory would be ever so slightly faster than reflection. I'm just not sure whether the slight performance improvement would be worth the extra complexity.
It's not working yet and I just didn't find the time to finish it.